app/models/case/sar/offender.rb

```diff
@@ -152,6 +176,7 @@ def searchable_fields_and_ranks
   validate :validate_partial_case_letter_sent_dated
   validate :validate_sent_to_sscl_at
   validate :validate_remove_sent_to_sscl_reason
+  validate :validate_offender_sar_rejected, if: -> { rejected? }
```

Comment:
This is on the offender sar model, so probably don't need to specify that in the method name.